tool/colors/update_colors.dart

```diff
@@ -56,14 +68,19 @@ List<String> extractColorNames(File file) {
   return Set<String>.from(names).toList()..sort();
 }
 
-void generateDart(List<String> colors, String colorType, String className) {
+void generateDart(
+  List<String> colors,
+  String colorType,
+  String className,
+  String baseUri,
+) {
   final StringBuffer buf = StringBuffer();
   buf.writeln('''
 // Generated file - do not edit.
 
 import 'dart:ui';
 
-import 'package:flutter/src/$colorType/colors.dart';
+import '$baseUri';
```

Comment:
nit: Let's name this `importUri` instead. I thought we'd be appending `/colors.dart` or similar on to it when I picked `baseUri` but realise now that was not correct. Thanks!